### ISSUE TITLE
[3.3.5/Vendor] Buyback need stay after log off

### ISSUE THREAD

**Issue by Keader** [TrinityCore/TrinityCore#23698]
**Description:** Items in Buyback aba, need stay after player log off.

Patch 3.3.3: Items sold to vendors now stay in the buyback page even if you log off.

Source: https://wowwiki.fandom.com/wiki/Patch_3.3.3_(undocumented_changes)

**Current behaviour:** After character log off, all items in Buyback table vanish

**Branch(es):**  3.3.5

**TC rev. hash/commit:** 31b4b897e4938b3f04d53c584b02ab4402132864

**Comment by illfated** [comment]
Good point, I had completely forgotten that this feature was available on retail in 2010.

**Comment by imstuckbro** [comment]
f08b08e7cb3796c2646c1ddb18bdee515217dd80 Bump, I guess this will need a separate db table (to store previously up to a [12 maximum](https://worldofwarcraft.fandom.com/et/wiki/Vendor) last sold items for each character).
